### DISCUSSION TITLE
Img src based on current theme?

### DISCUSSION THREAD

**Discussion by ptfw** [saadeghi/daisyui#1088] [General]
Hi @saadeghi ,
thanks again for all of your hard work creating and maintaining this excellent DaisyUI project! 
I searched around and did not find an answer to this question so bringing it here:

I have several images on my pages that I would like to have a light version and a dark version and show the appropriate version based on the currently selected DaisyUI theme. Any ideas on best approach for ths?

**Comment by saadeghi** [comment]
This is tricky but possible.  
Example: https://play.tailwindcss.com/drFiOtcokc

You can hide an image by default and then show it based on the parent's `data-theme` value, with a class name like this:
```css
class="hidden [[data-theme=light]_&]:block"
```

Let me know if you have any questions
